examples/servers/reading-list/.kube.yml

```diff
@@ -0,0 +1,97 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: reading-list-server
+spec:
+  minReadySeconds: 30
+  replicas: 5
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 0
+      maxSurge: 2
+  template:
+    metadata:
+      labels:
+        app: reading-list-server
+    spec:
+      terminationGracePeriodSeconds: 60
+      volumes:
+      - name: nyt-reading-list-tls 
+        secret:
+          secretName: nyt-reading-list-tls
+      containers:
+        - name: esp
+          image: "gcr.io/endpoints-release/endpoints-runtime:1.3.0"
+          args: [
+            "-S", "443",
```

Comment:
I'd just be hesitant to provide a non-working or incomplete kube.yml file without at least a big warning about what's missing and how to fill in those missing pieces.

Also, letsencrypt (and lego) is just one way to do this. For example, we terminate SSL at the ingress LB and then it's http from there on (but we also don't expose anything gRPC to the world).